stream-video-android-previewdata/src/main/kotlin/io/getstream/video/android/mock/StreamPreviewDataUtils.kt

```diff
@@ -143,16 +143,11 @@ public val previewMemberListState: List<MemberState>
         val participants = arrayListOf<MemberState>()
         previewCall.state.clearParticipants()
         previewUsers.forEach { user ->
-            val sessionId = if (user == previewUsers.first()) {
-                previewCall.sessionId
-            } else {
-                UUID.randomUUID().toString()
-            }
             participants.add(
                 MemberState(
                     user = user,
-                    createdAt = OffsetDateTime.now(),
-                    updatedAt = OffsetDateTime.now(),
+                    createdAt = OffsetDateTime.MIN,
+                    updatedAt = OffsetDateTime.MIN,
```

Comment:
I changed this as we had flakiness due to it, even tho I'm still not 100% sure about the root problem.

Basically, `.now()` will cause TreeTenABP to try loading from assets and this resulted in a `IllegalStateException: Asset repository is not set` <- how this fails (intermittently in CI) is the part I'm not sure about. In any case, we now avoid `.now()` so we remove the problem at the root.